backend/webserver/tests.py

```diff
@@ -1728,3 +1728,376 @@ def test_is_remote_post_only_allows_all_local_requests(self):
+        self.assertEqual(response.data[0]['post'],'http://testserver/api/authors/1/posts/1/')
+        self.assertEqual(response.data[1]['post'],'http://testserver/api/authors/1/posts/2/')
+    
+    def test_get_liked_public_posts_only(self):
+        author_1 = Author.objects.create(username="author_1", display_name="author_1")
+        author_2 = Author.objects.create(username="author_2", display_name="author_2")
+        author_3 = Author.objects.create(username="author_3", display_name="author_3")
+        post_1 = Post.objects.create(
+            author =author_2,
+            title="Post 1",
+            description="Testing post",
+            visibility= "PUBLIC",
+            content="Testing Post 1"
+        )
+        post_2 = Post.objects.create(
+            author=author_2,
+            title="Post 2",
+            description="Testing post 2",
+            visibility="PRIVATE",
+            content="Testing post 2"
+        )
+        post_3 = Post.objects.create(
+            author=author_3,
+            title="Post 3",
+            visibility="FRIENDS",
+            description="Testing post 3"   
+        )
+        post_4 = Post.objects.create(
+            author =author_2,
+            title="Post 4",
+            description="Testing post 4",
+            content="Some content",
+            visibility= "PUBLIC"
+        )
+        post_5 = Post.objects.create(
+            author =author_2,
+            title="Post 5",
+            description="Testing post 5",
+            content="Some content",
+            visibility= "PRIVATE"
+        )
+
+        Like.objects.create(liker=author_1,post=post_1)
+        Like.objects.create(liker=author_1,post=post_2)
+        Like.objects.create(liker=author_1,post=post_3)
+        Like.objects.create(liker=author_1,post=post_4)
+        Like.objects.create(liker=author_1,post=post_5)
+        self.assertEqual(5,Like.objects.count())
+        url = f'/api/authors/{author_1.id}/liked/'
+        self.client.force_authenticate(user=mock.Mock())
+        response = self.client.get(url,format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data), 2)
+        self.assertEqual(response.data[0]['liker']['id'],author_1.id)
+        self.assertEqual(response.data[1]['liker']['id'],author_1.id)
+        self.assertEqual(response.data[0]['post'],'http://testserver/api/authors/1/posts/1/')
+        self.assertEqual(response.data[1]['post'],'http://testserver/api/authors/1/posts/4/')
+
+    def test_no_posts_liked(self):
+        author_1 = Author.objects.create(username="author_1", display_name="author_1")
+        url = f'/api/authors/{author_1.id}/liked/'
+        self.client.force_authenticate(user=mock.Mock())
+        response = self.client.get(url,format="json")
+        self.assertEqual(len(response.data),0)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+class LikePostViewTestCase(APITestCase):
```

Comment:
When you change the names of the views as suggested above, please also remember to change these names